src/Microsoft.AspNetCore.Mvc.Abstractions/Microsoft.AspNetCore.Mvc.Abstractions.csproj

```diff
@@ -1,6 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <Import Project="..\..\build\common.props" />
+  <Import Project="..\..\build\sourcelink.props" />
```

Comment:
Haven't played with it. But, I'd start with
``` xml
<ItemGroup>
  <OtherProjectsToPack Include="@(ProjectsToPack)" Exclude="$(ProjectName)" />
</ItemGroup>
<Import Project="..\..\build\sourcelink.props" Condition=" @(OtherProjectsToPack) != @(ProjectsToPack) "/>
```

Of course, the path to `sourcelink.props` would be different when in `common.props`.